src/components/side-panel/side-panel-header.tsx

```diff
@@ -53,9 +54,7 @@ export const SidePanelHeader = () => {
             return (
                 <>
                     <SaveChangesButton>
-                        {focusedElement.isDirty === false
-                            ? "Saved"
-                            : "Save Changes"}
+                        {isDirty === false ? "Saved" : "Save Changes"}
```

Comment:
instead of checking for `false`, check for `falsy` to cover undefined / null values

```suggestion
                        {!isDirty ? "Saved" : "Save Changes"}
```

Reply:
fixed in the last push

Reply:
did you miss pushing this change?